community/contributing/pr_workflow.rst

```diff
@@ -125,30 +125,31 @@ By default, the ``git clone`` should have put you on the *master* branch of
 your fork (*origin*). To start your own feature development, we will create
 a feature branch:
 
-::
+.. code-block:: none
```

Comment:
I'd prefer that you remove ` .. highlight:: shell` instead of hardcoding `none` everywhere.

Reply:
Oh I thought the standard is automatic language detection, I missed this line :)